gems/sorbet/test/snapshot/run_one_bazel.sh

```diff
@@ -0,0 +1,176 @@
+#!/bin/bash
+
+# Runs a single srb init test from gems/sorbet/test/snapshot/{partial,total}/*
+
+shopt -s dotglob
```

Comment:
Do you think it's possible to put `set -euo pipefail` down below the runfiles.bash dance?